test/batches_SUITE.erl

```diff
@@ -124,7 +121,7 @@ aggregation_might_produce_noop_requests(_) ->
     {ok, Server} = gen_server:start_link(?MODULE, [], []),
     Requestor = fun(1, _) -> timer:sleep(1), gen_server:send_request(Server, 1);
                    (_, _) -> drop end,
-    Opts = (default_aggregator_opts(Server))#{pool_id => ?FUNCTION_NAME,
+    Opts = (default_aggregator_opts(Server))#{pool_id => test_pool,
```

Comment:
I know the pools don't **need** different id's, but why change it?

Reply:
It was causing errors in the test suite because the events were executed with an unregistered `pool_id` label